.travis.yml

```diff
@@ -1,4 +1,19 @@
+sudo: false
```

Comment:
only for new repos AFAIK

https://travis-ci.org/ember-cli/core-object/jobs/117546176 apparently ran on the "old" infrastructure